spec/spec_helper.rb

```diff
@@ -30,3 +34,5 @@ def fixtures_dir
     end
   end
 end
+
+RSpec::Matchers.define_negated_matcher :excluding, :include
```

Comment:
I think we don't need this one anymore, right?

Reply:
Good spot!